ansible/roles/edu-sharing/templates/mirror-vocabs.sh.j2

```diff
@@ -3,4 +3,8 @@
 {% for mirror in mirror_vocabularies %}
 wget -q --mirror "{{ mirror.url }}" -P {{ mirror.target_directory }}
 chown -R www-data:www-data {{ mirror.target_directory }}
+{% if ((mirror.isSorted | default("false") == 'true') and (mirror.createVariant | default("false") == 'true')) %}
+path={{ mirror.url [8:] | dirname }}
+jq --raw-output  '.hasTopConcept | map(.) | sort_by(.prefLabel.de) | "{hasTopConcept:\(.)}"' {{ mirror.target_directory }}/$path/scheme.json > {{ mirror.target_directory }}/$path/scheme_sort.json
```

Comment:
Hier wird ein fester Werte `scheme.json` für den Dateinamen verwendet. Das muss so aber nicht existieren. Wenn Du zB in die edu-sharing Dokumentation schaust: https://docs.edu-sharing.com/confluence/edp/de/configuration-en/metadatensets-suche-rendering-eingabemasken

Dort wird als Beispiel für ein SKOS-Vokabular folgende URL aufgeführt: https://vocabs.openeduhub.de/w3id.org/openeduhub/vocabs/widgets/index.json

=> also kein scheme.json

Bitte den Dateinamen aus der URL verwenden. Das geht mit ansible ganz leicht via `| basename` - siehe auch

https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#managing-file-names-and-path-names